SConstruct

```diff
@@ -52,7 +52,7 @@ DESIRED_FLAGS = ['-Wno-gnu-designator',
                  '-Wno-gnu-zero-variadic-macro-arguments',
                  '-Wno-tautological-constant-out-of-range-compare']
 
-CART_VERSION = "1.4.0"
+CART_VERSION = "1.4.1"
```

Comment:
New API should be minor version (e.g. 1.5)

Reply:
good point